src/karmen_backend/openapi/swagger.yaml

```diff
@@ -0,0 +1,1544 @@
+      operationId: server.routes.organizations.list_organizations
+      tags:
+        - organizations
+      responses:
+        '200':
+          description: 'List of organizations'
+  '/organizations/{org_uuid}':
+    patch:
+      summary: 'Reanme orgaization'
+      operationId: server.routes.organizations.update_organization
+      requestBody:
+        content:
+          application/json:
+            schema:
+              properties:
+                name:
+                  type: string
+      tags:
+        - organizations
+      parameters:
+        - $ref: '#/components/parameters/organization_uuid_param'
+      responses:
+        '200':
+          description: 'Renamed'
+  '/organizations/{org_uuid}/printjobs':
+    post:
+      summary: 'Start printing gcode on a printer'
+      operationId: server.routes.printjobs.printjob_create
+      tags:
+        - printjobs
+      parameters:
+        - $ref: '#/components/parameters/organization_uuid_param'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              properties:
+                gcode:
+                  type: string
+                  format: uuid
+                printer:
+                  type: string
+                  format: uuid
+      responses:
+        '201':
+          description: 'Printjob scheduled'
+        '409':
+          description: Could ot schedule a printjob
+    get:
+      summary: 'List printjobs'
+      operationId: server.routes.printjobs.printjobs_list
+      tags:
+        - printjobs
+      parameters:
+        - $ref: '#/components/parameters/organization_uuid_param'
+      responses:
+        '200':
+          description: 'List of printjobs'
+  '/organizations/{org_uuid}/printjobs/{printjob_uuid}':
+    get:
+      summary: 'Get printjob detail'
+      operationId: server.routes.printjobs.printjob_detail
+      tags:
+        - printjobs
+      parameters:
+        - $ref: '#/components/parameters/organization_uuid_param'
+        - $ref: '#/components/parameters/printjob_uuid_param'
+      responses:
+        '200':
+          description: 'Printjob detail'
+  '/organizations/{org_uuid}/tasks':
+    post:
+      summary: 'Schedule a task for organization'
+      operationId: server.routes.tasks.enqueue_task
+      requestBody:
+        content:
+          application/json:
+            schema:
+              properties:
+                task:
+                  type: string
+                  enum: [scan_network]
+      tags:
+        - tasks
+      parameters:
+        - $ref: '#/components/parameters/organization_uuid_param'
+      responses:
+        '202':
+          description: 'Task created'
+        '400':
+          description: Invalid task
+        '500':
+          description: Not allowed in clud mode
+
+
+
+
+
+components:
+  parameters:
+    printer_uuid_param:
+      name: printer_uuid
+      in: path
+      required: true
+      schema:
+        type: string
```

Comment:
OpenAPI can validate `uuid` when format is specified, see https://swagger.io/docs/specification/data-models/data-types/ and add format requirement where applicable.